src/Elasticsearch.Net/Extensions/StringBuilderCache.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+
+namespace Elasticsearch.Net.Extensions
+{
+	/// <summary>Provide a cached reusable instance of stringbuilder per thread.</summary>
+	internal static class StringBuilderCache
```

Comment:
This would be good to have in the codebase going forward.

Need to make sure our usages of `Acquire` don't overlap before being done with the `StringBuilder` though